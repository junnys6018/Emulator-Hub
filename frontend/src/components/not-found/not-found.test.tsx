import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from './not-found';

test('<NotFound /> snapshot', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
});
