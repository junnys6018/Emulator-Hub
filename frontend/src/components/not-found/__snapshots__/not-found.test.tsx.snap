// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> snapshot 1`] = `
<div
  className="text-center"
  style={
    Object {
      "marginBottom": "20vh",
      "marginTop": "20vh",
    }
  }
>
  <p
    className="font-semibold text-6xl md:text-7xl mb-8 md:mb-11"
  >
    Oops
  </p>
  <p
    className="font-medium text-gray-300 text-3xl md:text-4xl mb-4 md:mb-5"
  >
    Something Went Wrong
  </p>
  <p
    className="font-normal text-gray-300 text-xl md:text-2xl"
  >
    Error 404 Page Not Found
  </p>
</div>
`;
